MVCapplication/src/main/webapp/WEB-INF/resources/js/analytics.js

```diff
@@ -20,3 +17,110 @@ $("#selectOtherProduct").click(function () {
     location.href = $(this).attr("href");
 });
 
+
+/**
+ * This block of code gets data from .jsp and displays charts of using and purchasing products
+ */
+
+$(document).ready( function() {
+
+    var purchasingProductDatesForChart = getArray($("#purchasingProductDatesForChart"));
+    var purchasingProductAmountsForChart = getArray($("#purchasingProductAmountsForChart"));
+    var usingProductDatesForChart = getArray($("#usingProductDatesForChart"));
+    var usingProductAmountsForChart = getArray($("#usingProductAmountsForChart"));
+    var endDate = $("#endDate").text().trim();
+    console.log(endDate);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2228)